frontend/js/src/explore/artist-similarity/artist-search/SearchDropdown.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { ArtistType } from "./artistLookup";
+import SearchItem from "./SearchItem";
+
+interface SearchDropdownProps {
+  searchResults: Array<ArtistType>;
+  onArtistChange: (artist: string) => void;
+  id: string;
+  onDropdownChange: (openDropdown: boolean) => void;
+}
+
+function SearchDropdown({
+  searchResults,
+  onDropdownChange,
+  onArtistChange,
+  id,
+}: SearchDropdownProps) {
+  return (
+    <div id={id}>
```

Comment:
You don't need to receive the `id` as a prop. Also, if there are multiple usages of this component you may want to change this to a class.

Reply:
Fixed